test/integration/transactions_test.go

```diff
@@ -863,3 +865,40 @@ func TestAccountPropertiesEntityTypeTransaction(t *testing.T) {
 	}, testAccount)
 	assert.Nil(t, result.error)
 }
+
+func TestModifyMosaicLevyTransaction(t *testing.T) {
+
+	// Add levy to XPX mosaic
+	mosaicId, _ := sdk.NewMosaicId(XPXID)
+
+	result := sendTransaction(t, func() (sdk.Transaction, error) {
+		return client.NewMosaicModifyLevyTransaction(
+			sdk.NewDeadline(time.Hour),
+			mosaicId,
+			sdk.MosaicLevy{
+				Type: sdk.Levy_PercentileFee,
+				// supply valid address here for testing
+				Recipient: sdk.NewAddress("SBGVTUFYMSFCNHB2SO33C54UKLFBJAQ5457YSF2O", client.NetworkType()),
+				Fee: sdk.CreateMosaicLevyFeePercentile(1.5),
```

Comment:
Do we have test case for fixed amount?

We also need to test the fail-safe scenarios such: no xpx available, not enough xpx, not enough xpx to pay for fees.

Reply:
Fail scenarios is not part of SDK's tests, it is part of acceptance tests=)